app/_src/policies/introduction.md

```diff
@@ -1,6 +1,650 @@
+
+#### Example
+
+The following policy will only apply to gateway data-planes:
+{% policy_yaml proxytypes %}
+```yaml
+type: MeshTimeout
+name: gateway-only-timeout
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+    proxyTypes: ["Gateway"]
+  to:
+    - targetRef:
+        kind: Mesh
+      default:
+        idleTimeout: 10s
+```
+{% endpolicy_yaml %}
+
+### Target resources
+
+Not every policy supports `to` and `from` levels. Additionally, not every resource can
+appear at every supported level. The specified top level resource can also affect which
+resources can appear in `to` or `from`.
+
+To help users, each policy documentation includes tables indicating which `targetRef` kinds is supported at each level.
+For each type of proxy, sidecar or builtin gateway, the table indicates for each
+`targetRef` level, which kinds are supported.
+
+#### Example tables
+
+These are just examples, remember to check the docs specific to your policy!
+
+{% tabs targetRef useUrlFragment=false %}
+{% tab targetRef Sidecar %}
+| `targetRef`             | Allowed kinds                                            |
+| ----------------------- | -------------------------------------------------------- |
+| `targetRef.kind`        | `Mesh`, `MeshSubset`, `MeshService`                      |
+| `to[].targetRef.kind`   | `Mesh`, `MeshService`                                    |
+| `from[].targetRef.kind` | `Mesh`                                                   |
+{% endtab %}
+
+{% tab targetRef Builtin Gateway %}
+| `targetRef`           | Allowed kinds                                    |
+| --------------------- | ------------------------------------------------ |
+| `targetRef.kind`      | `Mesh`, `MeshGateway`, `MeshGateway` with `tags` |
+| `to[].targetRef.kind` | `Mesh`                                           |
+{% endtab %}
+{% endtabs %}
+
+#### Sidecar
+
+We see that we can select sidecar proxies via any of the kinds that select
+sidecars and we can set both `to` and `from`.
+
+We can apply policy to:
+* all traffic originating at the sidecar _to_ anywhere (`to[].targetRef.kind: Mesh`)
+* traffic _to_ a specific `kuma.io/service` (`to[].targetRef.kind: MeshService`)
+
+We can also apply policy to:
+* traffic terminating at the sidecar _from_ anywhere in the mesh (`from[].targetRef.kind: Mesh`)
+
+#### Builtin gateways
+
+We see that we can select gateway proxies via any of the kinds that select
+gateways as well as specific gateway listeners and we can set only `to`.
+
+We can only apply policy to:
+* all traffic originating at the gateway _to_ anywhere (`to[].targetRef.kind: Mesh`)
+
+## Merging configuration
+
+It is necessary to define a policy for merging configuration,
+because a proxy can be targeted by multiple `targetRef`'s.
+
+We define a total order of policy priority:
+
+- MeshServiceSubset > MeshService > MeshSubset > Mesh (the more a `targetRef` is focused the higher priority it has)
```

Comment:
What does it mean for targetRef to be more focused? Maybe: `the more broad targetRef is, the lower the priority it has`